pkg/yamltemplate/filetests/ytt-library/assert-equals.yml

```diff
@@ -0,0 +1,10 @@
+#@ load("@ytt:assert", "assert")
+#@ load("@ytt:json", "json")
+
+equals_works: #@ assert.equals("foo", "foo")
+
++++
+
+equals_works:
+- null
```

Comment:
Thinking about the signature of this function. Currently, when the two parameters are equal it returns null. Maybe we want to return True instead, to indicate success. This is a bit different than other equals functions since if the result is ever false, it will error out instead of return False. Still, maybe returning True could be helpful here.

Reply:
I put more thought into this and realized that two return types (returning a tuple) are not needed, and rather than return a tuple or True, we can just return starlark.None. 

In the success case: return starlark.None. This is essentially a no-op.

In the error case: return the error. Returning an error from a Starlark builtin function, will stop execution and report the error. So there is no need to return an error for the user to check, since execution is stopped.

Then this results in this tests now only needs to check a single `null` value is returned.